app/Http/Controllers/HomeController.php

```diff
@@ -29,7 +30,9 @@ public function index()
 
     public function create()
     {
+        $cart = Cart::all();
         $product = Product::all();
-        return view('welcome', compact('product'));
+        $counters = Cart::all('product_id')->count();
+        return view('welcome', compact('product', 'cart', 'counters'));
```

Comment:
Always give your code spaces, especially before you return anything... Like where you are returning to view

Reply:
owk sir, Noted!